src/library-authoring/create-library/data/apiHooks.ts

```diff
@@ -0,0 +1,22 @@
+// @ts-check
```

Comment:
You can remove `// @ts-check` from .ts files.

Reply:
Done 8d712972dc53e00ef9d411a1d7b3ff919a53027a!